### DISCUSSION TITLE
Having issues with Truffle migrate.

### DISCUSSION THREAD

**Discussion by OGXCHANGE** [ConsenSys-archive/truffle#5887] [General Q&A]
Trying to migrate my contract but not going according to plan. getting this error
 "TypeError: tether.transfer is not a function
    at module.exports (C:\Users\user\defi-staking-app-starter\defi-staking-app-starter\migrations\2_deploy_contracts.js:23:16)
    at process._tickCallback (internal/process/next_tick.js:68:7)
Truffle v5.1.39 (core: 5.1.39)
Node v10.15.3" 
below is the 2_deploy_contracts.js file
const RWD = artifacts.require('RWD')
const Tether = artifacts.require('Tether')
const DecentralBank = artifacts.require('DecentralBank')

module.exports = async function(deployer, network, accounts) {
  
  // Deploy Mock Tether Token
  await deployer.deploy(Tether)
  const tether = await Tether.deployed()
 
  // Deploy RWD Token
  await deployer.deploy(RWD)
  const rwd = await RWD.deployed()
  
  // Deploy DecentralBank
  await deployer.deploy(DecentralBank, rwd.address, tether.address)
  const decentralBank = await DecentralBank.deployed()

  // Transfer all RWD tokens to Decentral Bank
  await rwd.transfer(decentralBank.address, '1000000000000000000000000')

  //distribute 100 Tether to investor
  await tether.transfer(accounts[2], '1000000000000000000000000')
 };

**Comment by cds-amal** [comment]
Do you have an repo you can share? It's difficult to assist otherwise. If you do have a repo, please be careful to not share your mnemonic, private keys or other secret credentials.

**Reply by OGXCHANGE** [reply]
thanks for the reply, i have been watching youtube videos to see how i can push my files to the repo but i dont seem to get it, First is that the project i am working on was cloned but i have added some other folders and files but dont seem to get a hang of it so i push it to my repo.

**Reply by OGXCHANGE** [reply]
I have been able to sort this out, just discovered that the fault is from visual studio. i didnt know that it has not been saving all this while. thank you guys.
